File: Takseer.py
def transform_string(input_string):
    input_string = input_string.replace(" ","")
    result = ""
    length = len(input_string)
    last = length - 1
    if length % 2 == 0:
        for i in range(length // 2):
            result += input_string[i]
            result += input_string[last]
            last -= 1
    else:
    # if length % 2 == 1:
        length = length - 1
        for i in range(length // 2):
            result += input_string[i]
            result += input_string[last]
            last -= 1
            if i == length // 2 - 1:
                result += input_string[i + 1]
    arr.append(result)
    print(result)
    if result != check:
        transform_string(arr[-1])
arr = []
input_string = input("enter: ")
arr.append(input_string)
check = arr[-1].replace(" ", "")
transform_string(arr[-1])

File: Takseer.js
function takseer(string) {
  let result = ""
  string = string.replaceAll(" ", "")
  let stringLength = string.length
  let last = stringLength - 1
  if (stringLength % 2 == 0) {
    for (let i = 0; i < stringLength / 2; i++) {
      result = result.concat(string[i])
      result = result.concat(string[last])
      last--
    }
  } else {
    stringLength = stringLength - 1
    for (let i = 0; i < stringLength / 2; i++) {
      result = result.concat(string[i])
      result = result.concat(string[last])
      last--
      if (i == stringLength / 2 - 1) {
        result = result.concat(string[i + 1])
      }
    }
  }
  arr.push(result)
  console.log(result)
  if (result !== check) {
    takseer(arr.at(-1))
  }
}
const arr = []
var string = prompt("Enter value")
arr.push(string)
let arr1 = arr[0]
var check = arr1.replaceAll(" ", "")
takseer(arr.at(-1))